packages/charts/src/renderers/radial/gauge/donut-gauge-renderer-config.ts

```diff
@@ -7,7 +7,6 @@ import { IRadialRendererConfig } from "../../../core/common/types";
  */
 export function donutGaugeRendererConfig(): IRadialRendererConfig {
     return {
-        annularGrowth: 0,
```

Comment:
@elenates, the annularGrowth override of zero has been removed for the gauge renderer so the gauge thickness will grow at the same rate as a standard donut chart.